osu.Game/Rulesets/Mods/MetronomeBeatContainer.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Audio.Track;
+using osu.Framework.Graphics;
+using osu.Game.Audio;
+using osu.Game.Beatmaps.ControlPoints;
+using osu.Game.Graphics.Containers;
+using osu.Game.Skinning;
+
+namespace osu.Game.Rulesets.Mods
+{
+    public class MetronomeBeatContainer : BeatSyncedContainer
```

Comment:
Probably just rename to `Metronome`. Not sure about this class being in the mods ruleset, as there's no other similar "helper" classes in there, but also unsure on a better place to put it.

Reply:
I think it's fine to put the class in the mods folder. `DifficultyBindable` and `DifficultyAdjustSettingsControl` are already there, so this won't be the first mods-specific helper class in that folder.